app/src/main/java/com/example/travelapp/Dijkstras.java

```diff
@@ -70,28 +81,34 @@ public static Graph createGraph(List<String> ids, Itinerary itinerary) throws In
 
     public static List<Integer> getShortestPath(Graph graph, int src, int tgt) {
         List<Integer> list = new ArrayList<Integer>();
+        // hard coded base case of size = 2 since no edge between 0 and 1 is added in the graph.
+        if (graph.getSize() == 2) {
+            list.add(0);
+            list.add(1);
+            return list;
+        }
```

Comment:
imo this should be moved under the check for src == tgt since src == tgt is the real base case